randovania/bitpacking/construct_dataclass.py

```diff
@@ -34,12 +45,14 @@ def construct_for_dataclass(cls) -> construct.Construct:
 
 _direct_mapping = {
     bool: construct.Flag,
-    int: construct.VarInt,
+    int: construct.ZigZag,
```

Comment:
out of curiosity, was being unsigned causing any problems anywhere or are you just future proofing?

Reply:
there's certainly values that could be thrown around that are negative, though I guess PickupEntry doesn't use this rn